.github/workflows/android.yml

```diff
@@ -98,6 +98,14 @@ jobs:
           name: android_${{ matrix.type }}_${{ github.event.pull_request.head.sha }}
           path: deploy/android_${{ matrix.type }}/*.7z
 
+      - name: Test Report
+        uses: dorny/test-reporter@v1
+        if: (success() || failure()) && matrix.type == 'manager'   # run this step even if previous step failed
```

Comment:
You can remove `success() || failure()` if you want to have this step to be always executed

Reply:
Fix.